pandas/tests/frame/test_sorting.py

```diff
@@ -227,6 +227,21 @@ def test_stable_descending_multicolumn_sort(self):
                                    kind='mergesort')
         assert_frame_equal(sorted_df, expected)
 
+    @pytest.mark.parametrize('seed', list(range(100)))
+    def test_sort_multi_index(self, seed):
+        # GH 25775
+        pd.np.random.seed(seed)
+        df1 = (pd.DataFrame(pd.np.random.rand(10, 5),
```

Comment:
`DataFrame` is already imported, so you don't need the `pd.` prefix.

Reply:
thanks, updated